js/okx.js

```diff
@@ -1248,6 +1331,16 @@ module.exports = class okx extends Exchange {
         return result;
     }
 
+    getCommonNetworkNameFromId (networkId, currencyCode = undefined) {
```

Comment:
> isn't this basically "safeNetwork()"?

I'll reply.

Reply:
@carlosmiei that is not safeNetwork, for two reasons:
- the 'safeNetwork' name is too vague and should be totally removed from lib.
- the functionality is different. I will explain this in the end of the pushes (which are remaining) after that I'll turn this PR from draft to "Ready" and then I'll explain things.